actions/if_color_exists_MOD.js

```diff
@@ -0,0 +1,51 @@
+module.exports = {
+  name: 'Check if Hex Color Code Exists',
+  section: 'Color Stuff',
+  meta: {
+    version: '2.1.7',
+    author: 'DBM Mods',
+  },
+
+  subtitle(data, presets) {
+    return `Check if Hex Color Code is Valid: ${presets.getConditionsText(data)}`;
+  },
+
+  fields: ['colorCode', 'branch'],
+
+  html() {
+    return `
+<div style="float: left; width: 60%">
+  <span class="dbminputlabel">Hex Color Code</span>
+  <input id="colorCode" class="round" type="text" placeholder="Enter hex color code">
+</div>
+<br><br><br>
+
+<conditional-input id="branch" style="padding-top: 8px;"></conditional-input>`;
+  },
+
+  init() {},
+
+  async action(cache) {
+    const data = cache.actions[cache.index];
+    const colorCode = this.evalMessage(data.colorCode, cache);
+    let result;
+
+    if (isValidHexColorCode(colorCode)) result = true;
+    else result = false;
```

Comment:
Having a separate function to check if its valid isn't necessary if its only used once. You can still separate the regex variable for readability but I would recommend the following snippet of code.

```js
const result = /^#(?:[0-9a-fA-F]{3}){1,2}$/.test(colorCode);
```